tools/synchro.py

```diff
@@ -271,15 +271,26 @@ def translate_docstrings(lines: list[str]) -> list[str]:
                 # This sequence of replacements fixes the grammar issues caused by translating async -> sync
                 if "an Async" in lines[i]:
                     lines[i] = lines[i].replace("an Async", "a Async")
+                if "an 'Async" in lines[i]:
+                    lines[i] = lines[i].replace("an 'Async", "a 'Async")
```

Comment:
Why replace "an Async" with "a Async"? Isn't "an" the correct word here?

Reply:
This happens before we apply the replacements of async -> sync, so "A Asyncx" will become "A x".